webprojects/amo.md

```diff
@@ -0,0 +1,93 @@
+# AMO: how to localize and test
+
+## Product at a glance
+
+AMO refers to https://addons.mozilla.org/. It is one of Mozilla’s biggest web properties, with 12 million users per month. The product consists three key components that require localization:
+
+* [AMO server](https://pontoon.mozilla.org/projects/amo) is the backend component of AMO
+* [AMO frontend](https://pontoon.mozilla.org/projects/amo-frontend/) covers the top page of `about:addons` and the user-facing pages of addons.mozilla.org on Firefox for Android.
+* [Disvoery Pane](https://pontoon.mozilla.org/projects/amo-frontend/) is home of curated addons available on `about:addons`.
+
+Product updates are deployed every Thursday to production. Localizable strings can be extracted on Tuesdays, and changes landed by 5pm UTC each Tuesday go live on Thursday. To report issues, go to: https://github.com/mozilla/addons-server/issues/new.
+
+Now let’s take a look of these components in greater details.
+
+## AMO and AMO frontend
+
+### [addons-server](https://pontoon.mozilla.org/projects/amo)
+
+**Addons-server** is the default front-end for desktop users of AMO. It is also the API backend and home to the developer hub and reviewer tools.
+
+### [addons-frontend](https://pontoon.mozilla.org/projects/amo)
+
+**Addons-frontend** is the new frontend and default frontend for mobile user-agents. It will eventually replace the **addons-server** frontend for user-pages. **Addons-frontend** currently provides an AMO app (amo.po).
+
+### Test the server and the frontend
+
+The strings will be available on Dev or Staging server depending on when they were committed. See [this section](https://github.com/mozilla-l10n/localizer-documentation/blob/2f065c6a15208cf30ec0a8d440a399a5b5424d39/webprojects/amo.md#frequencies-of-newly-extracted-strings-and-landing-of-newly-localized-strings) for details.
```

Comment:
It's an internal relative link, not an external one: `See [this section](#frequencies-of-newly-extracted-strings-and-landing-of-newly-localized-strings) for details.`